charts/pgadmin4/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if and (.Values.serviceAccount.enabled) (not .Values.serviceAccount.name) }}
```

Comment:
Makes more sense if you would `create` a service account rather then enabling it. If not mistake this is also the default layout within helm3:

{{- if .Values.serviceAccount.create -}}